frontend/public/components/lang-selector.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
```

Comment:
We should remove the lang selector from the masthead before we merge and use the browser language settings insetad. At some point we can add our own selector, but that can be a follow on that needs design input.